senza/manaus/constants.py

```diff
@@ -0,0 +1,11 @@
+ELB_REGION_HOSTED_ZONE = {"us-east-1": "Z35SXDOTRQ7X7K",
```

Comment:
I think I found a way to get the hosted zone in runtime. I'm going to try to implement it.